src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -1,339 +1,263 @@
 package config
 
 import (
-	"crypto/x509"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"os"
-	"path/filepath"
-	"regexp"
-	"sort"
 	"time"
 
-	"github.com/getlantern/appdir"
-	"github.com/getlantern/fronted"
-	"github.com/getlantern/golog"
-	"github.com/getlantern/keyman"
-	"github.com/getlantern/proxiedsites"
-	"github.com/getlantern/yaml"
-	"github.com/getlantern/yamlconf"
-
 	"github.com/getlantern/flashlight/client"
 	"github.com/getlantern/flashlight/proxied"
-)
-
-const (
-	cloudfront = "cloudfront"
-
-	// DefaultUpdateServerURL is the URL to fetch updates from.
-	DefaultUpdateServerURL = "https://update.getlantern.org"
+	"github.com/getlantern/golog"
+	"github.com/getlantern/rot13"
+	"github.com/getlantern/tarfs"
+	"github.com/getlantern/yaml"
 )
 
 var (
 	log = golog.LoggerFor("flashlight.config")
-	m   *yamlconf.Manager
-	r   = regexp.MustCompile("\\d+\\.\\d+")
-)
 
-// Config contains general configuration for Lantern either set globally via
-// the cloud, in command line flags, or in local customizations during
-// development.
-type Config struct {
-	configDir             string
-	Version               int
-	CloudConfigCA         string
-	CPUProfile            string
-	MemProfile            string
-	UpdateServerURL       string
-	BordaReportInterval   time.Duration
-	BordaSamplePercentage float64
-	Client                *client.ClientConfig
-	ProxiedSites          *proxiedsites.Config // List of proxied site domains that get routed through Lantern rather than accessed directly
-	TrustedCAs            []*fronted.CA
-}
-
-// StartPolling starts the process of polling for new configuration files.
-func StartPolling() {
-	// No-op if already started.
-	m.StartPolling()
-}
-
-// validateConfig checks whether the given config is valid and returns an error
-// if it isn't.
-func validateConfig(_cfg yamlconf.Config) error {
-	cfg, ok := _cfg.(*Config)
-	if !ok {
-		return fmt.Errorf("Config is not a flashlight config!")
+	// GlobalURLs are the chained and fronted URLs for fetching the global config.
+	GlobalURLs = &ChainedFrontedURLs{
```

Comment:
`PipeConfig` accept `ChainedFrontedURLs` as a parameter, but the possible values are defined in the very same file. Consider move these constants up to `flashlight` package as a separate file like `config_urls.go`?